Index: django_learn/learning-django-2825501/wisdompets/adoptions/models.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from django.db import models\n\n# Create your models here.\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/django_learn/learning-django-2825501/wisdompets/adoptions/models.py b/django_learn/learning-django-2825501/wisdompets/adoptions/models.py
--- a/django_learn/learning-django-2825501/wisdompets/adoptions/models.py	(revision f532e4e86dcf18df7db9b29c56c29e11edfc3f52)
+++ b/django_learn/learning-django-2825501/wisdompets/adoptions/models.py	(date 1675983823810)
@@ -1,3 +1,19 @@
 from django.db import models
 
-# Create your models here.
+
+class Pet(models.Model):
+    SEX_CHOICES = [('M', 'Male'), ('F', 'Female ')]
+
+    name = models.CharField(maxlenght=100)
+    submitter = models.CharField(maxlenght=100)
+    species = models.CharField(maxlenght=100)
+    breed = models.CharField(maxlenght=100, blank=True)
+    description = models.TextField(maxlenght=100)
+    sex = models.CharField(choices=SEX_CHOICES, max_length=1, blank=True)
+    submission_date = models.DateTimeField()
+    age = models.IntegerField(null=True)
+
+    vaccinations = models.ManyToManyField('Vaccine', blank=True)
+
+class Vaccine(models.Model):
+    name = models.CharField(max_length=50)
